### ISSUE TITLE
Lanching with multiple camera_nodelet_manager gives erratic behavior 

### ISSUE THREAD

**Issue by Thomas-Merlet** [ros-drivers/flir_camera_driver#101]
Hi,

I've been struggling to launch multiple cameras using this node.

After some investigation, it appears that launching the node with multiple camera_nodelet_manager was causing the issue. 
Doing so, the ROS driver will either:
- Work as expected
- Ouptut one of the following errors 
` Failed to connect with error: [SpinnakerCamera::connect] Failed to connect to camera. Error: Spinnaker: Camera is not communicating in U3V protocol. [-1015]`
`[SpinnakerCamera::grabImage] Failed to retrieve buffer with error: Spinnaker: Failed waiting for EventData on NEW_BUFFER_DATA event. [-1011]`
- Make the entire PC crash on startup

This is unconsistent with the comment in the `stereo.launch` file (` Both cameras are not loaded into one nodelet manager to avoid the namespacing issue.`), algouth it's not was is done in this launch file. Launching multiple `camera.launch` causes the problem. Following the `stereo.launch` layout and expending it for multiple cameras solved the issue. 

**Details**:

- **flir_camera_driver**: 2.0.3
- **spinnaker** 2.5.0.80
- **OS**: Ubuntu 18.04 LTS (ROS Melodic)

Thanks

**Comment by MathiasStokholm** [comment]
@Thomas-Merlet I've seen this behavior when trying to run while a newer version of the Spinnaker SDK is installed. Can you try just uninstalling the SDK and reconfiguring + compiling?

**Comment by Thomas-Merlet** [comment]
@MathiasStokholm Yes, I've actually encountered this problem on multiple computer, with different installation. I was using Spinnaker SDK 2.5.0.80 for my tests.
